Sections/EEA/GPPExtension: IAB Europe TCF.md

```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
+  <td>1 Purpose 1 was NOT disclosed at all.<br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
   </tr>
   <tr>
   <td>PublisherCC</td>
   <td>String (2)</td>
-  <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
-    Each letter is encoded as 6 bits, <code>a=0..z=25</code>. <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2"> ISO 3166-1 alpha-2 code </a> </td>
+    <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
+    Each letter is encoded as 6 bits, <code>a=0..z=25</code>. <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2"> ISO 3166-1 alpha-2 code </a>
+    </td>
+  </tr>
+  <tr style="background-color:#000;color:#FFF;">
+  <td colspan="4">
+        <strong>Vendor Consent and Vendor Legitimate Interest Sections</strong>
+      </td>
   </tr>
   <tr>
-  <td>VendorConsent</td>
+  <td>VendorConsent*</td>
   <td>OptimizedIntRange</td>
-  <td>List of Vendor IDs that indicate Consent for these vendors. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is array of int.<br></br>Equivalent of the fields listed under the Vendor Consent Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a> <br></br>MaxVendorId, <br></br>IsRangeEncoding, <br></br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId</td>
+  <td>List of Vendor IDs that indicate Consent for these vendors. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is an array of int.<br><br>Equivalent of the fields listed under the Vendor Consent Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a> <br>MaxVendorId, <br>IsRangeEncoding,<br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId</td>
   </tr>
   <tr>
-  <td>VendorLegitimateInterest</td>
+  <td>VendorLegitimateInterest*</td>
```

Comment:
remove the *, the field will be returned by the API

Reply:
Fixed